### DISCUSSION TITLE
Using Sloeber with Arduino Giga R1 WiFi

### DISCUSSION THREAD

**Discussion by PineappleFruitDud** [Sloeber/arduino-eclipse-plugin#1657] [Q&A]
Hey is there a workaround for using Sloeber with the Arduino Giga R1 WiFi? I cannot find it in the device list. 
Thanks a lot!

**Comment by jantje** [comment]
Did you install the platform/board?
![image](https://github.com/user-attachments/assets/c73fe0ec-ec23-47d6-86e4-c25848b1ed92)

**Comment by PineappleFruitDud** [comment]
In fact I did not do this. Unfortunately it does not make a difference as the giga board still is not visible.

![grafik](https://github.com/user-attachments/assets/82a5a342-12e7-477f-ac3e-7d5ee07b0a09)

**Comment by PineappleFruitDud** [comment]
I just had to select a different platform folder...now it works. Sorry for my stupidness and thank you!

**Comment by jantje** [comment]
No worries. We have all been there.
